php_phongo_classes.h

```diff
@@ -270,6 +270,7 @@ extern zend_class_entry *php_phongo_executiontimeoutexception_ce;
 extern zend_class_entry *php_phongo_connectiontimeoutexception_ce;
 extern zend_class_entry *php_phongo_writeexception_ce;
 extern zend_class_entry *php_phongo_bulkwriteexception_ce;
+extern zend_class_entry *php_phongo_serverexception_ce;
```

Comment:
GitHub says that it can't merge this branch because of conflicts. That's likely due to the massive clang-format patch that I merged earlier this week. That patch moved for example the ``*`` in these ``extern zend_class_entry`` lines to sit behind ``zend_class_entry``: https://github.com/mongodb/mongo-php-driver/commit/0a5fd12e202e3e1dffec314e141527a092af8ba1#diff-6639867923f0928bc37a3c06188c3401R300

Could you please rebase your branch on the master branch, and then run ``make format`` (after running ``phpize`` and ``./configure --enable-developer-flags``) ? The ``make format`` should also (re-)format the code according to clang format's rules.